pkg/runtime/reconciler.go

```diff
@@ -271,10 +271,17 @@ func (r *resourceReconciler) Sync(
 		return err
 	}
 	for _, condition := range latest.Conditions() {
-		if condition.Type == ackv1alpha1.ConditionTypeResourceSynced &&
-			condition.Status != corev1.ConditionTrue {
-			return requeue.NeededAfter(
-				ackerr.TemporaryOutOfSync, requeue.DefaultRequeueAfterDuration)
+		if condition.Type == ackv1alpha1.ConditionTypeResourceSynced {
+			if condition.Status == corev1.ConditionTrue {
+				duration, status := r.rmf.GetRequeueOnSuccessSeconds()
```

Comment:
nit: can you rename 'status' to 'reque'? because there is already a status in this code block

Reply:
done